src/component/handlers/edit/getFragmentFromSelection.js

```diff
@@ -19,12 +19,15 @@ import type EditorState from 'EditorState';
 
 function getFragmentFromSelection(editorState: EditorState): ?BlockMap {
   var selectionState = editorState.getSelection();
-  if (!selectionState.isCollapsed()) {
-    return getContentStateFragment(
-      editorState.getCurrentContent(),
-      selectionState
-    );
+
+  if (selectionState.isCollapsed()) {
+    return undefined;
```

Comment:
Can we make this `null` too? It looks like we only check truthiness of this value before indexing into it.